.travis.yml

```diff
@@ -1,47 +1,72 @@
-
+sudo: false
+os:
+  - linux
+  - osx
```

Comment:
I'm not entirely sure that "sets default and picks base image" is true when the os is `osx` - I think there is only one `osx` image, so... I don't agree with your ordering reasoning. 
However, I also don't care which order these things are in, and since you do (likely because you haven't spent the last two weeks looking at this stupid file) I'll happily move it :)